src/applications/claims-status/tests/utils/helpers.unit.spec.js

```diff
@@ -471,10 +471,9 @@ describe('Disability benefits helpers: ', () => {
       const contents = getStatusContents(type, details);
       expect(contents.title).to.equal('Your hearing has been scheduled');
       // TO-DO: Update with real content
-      const descText = shallow(contents.description)
-        .render()
-        .text();
-      expect(descText).to.contain(expectedDescSnippet);
+      const descText = shallow(contents.description);
+      expect(descText.render().text()).to.contain(expectedDescSnippet);
+      descText.unmount();
```

Comment:
this was another type of minor refactor- just moved the SUT querying into the expect statement